.github/workflows/tag-to-draft-release.yml

```
@@ -25,7 +28,8 @@ jobs:
     - name: Set GIT tag name
       run: |
         # Sets an environment variable used in the next steps
-        echo "TRAVIS_TAG=$(git describe --exact-match --tags)" >> $GITHUB_ENV
+        TRAVIS_TAG=$(git describe --exact-match --tags)"
```

```suggestion
        TRAVIS_TAG=$(git describe --exact-match --tags)
```
(I typed this wrong)

---

Thanks for reviewing (I didn't make a copy-paste, It's simply that I made the same mistake)